migration_steps/transform_casrec/transform/app/entities/finance_order/__init__.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from helpers import log_title, check_entity_enabled
+from entities.finance_order.finance_order import insert_finance_order
+
+log = logging.getLogger("root")
+
+
+def runner(target_db, db_config):
+    """
+    | Name          | Running Order | Requires |
+    | --------------| ------------- | -------- |
+    | finance_order | 1             |          |
+    """
+
+    entity_name = "finance_order"
+    extra_entities = []
```

Comment:
Does this have no dependencies at all? This check can be removed if not.

Reply:
Good point, it does depend on cases